crates/tako/src/internal/messages/worker.rs

```diff
@@ -40,6 +40,8 @@ pub struct ComputeTaskMsg {
 
     #[serde(with = "serde_bytes")]
     pub body: Box<[u8]>,
```

Comment:
Do we still need the body here?

Reply:
We need it because task sharing works now only in the server. So for workers we are sending the body for each task.